rules/add-a-description-to-github-repositories/rule.md

```diff
@@ -0,0 +1,27 @@
+---
+type: rule
+archivedreason: 
+title: Do use spaces in table names?
+guid: 1c244aed-36dd-48c5-9232-1b02f2ca90e3
+uri: add-a-description-to-github-repositories
+created: 2021-05-19T05:09:16.0000000Z
+authors:
+- title: Brady Stroud
+  url: https://ssw.com.au/people/brady-stroud
+related:
+- do-you-know-how-to-name-a-github-repository
+redirects: []
+
+---
+
+It is important to add a description to your repositories so they do not appear blank in the preview on your profile.
```

Comment:
Can we say - 'On your Github profile'. 

For non-technical people like me, we might not be as familiar that we actually have a profile when we make rule changes (only really ever navigate to my profile when get told to).